setup.py

```diff
@@ -37,9 +37,9 @@ def get_version(rel_path):
     packages=['upcloud_api', 'upcloud_api.cloud_manager'],
     download_url='https://github.com/UpCloudLtd/upcloud-python-api/archive/%s.tar.gz' % version,
     license='MIT',
-    python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
+    python_requires='>3.5',
```

Comment:
I know that Python 2.7 had it's EOL this year on january, but could we still support it, since it's the last python 2 version? If there is no conflicts i think it would be great to be able to support it still.

Reply:
Sure. 2.7 had no problems with updated and new packages, I will update the pr.